src/modules/Modal/ModalActions.js

```diff
@@ -30,11 +39,20 @@ ModalActions.propTypes = {
   /** An element type to render as (string or function). */
   as: customPropTypes.as,
 
+  /** Elements to render as Modal action buttons. */
+  actions: PropTypes.arrayOf(customPropTypes.itemShorthand),
+
   /** Primary content. */
   children: PropTypes.node,
 
   /** Additional classes. */
   className: PropTypes.string,
 }
 
+ModalActions.defaultProps = {
+  actions: [],
+}
+
+ModalActions.create = createShorthandFactory(ModalActions, actions => ({ actions }), true)
```

Comment:
Because there will only be one instance of Modal.Actions, the Modal.Actions component itself won't need to have keys.  We can remove `true` from here.

See above comment regarding `Button.create` which is what will need the keys, since, there is an array of button elements being created (within Modal.Actions).